lms/templates/instructor/instructor_dashboard_2/rapid_response.html

```diff
@@ -0,0 +1,25 @@
+<%page args="section_data" expression_filter="h"/>
+<%!
+from django.urls import reverse
+from itertools import groupby
+from django.utils.translation import ugettext as _
+%>
+
+
+<div>
+    % for date,runs in groupby(section_data['problem_runs'],key=lambda x:x['created'].date()):
+    <ul>
+        <li>${date.strftime('%Y/%m/%d')}</li>
+        <ul>
+            % for run in runs:
+            <li>${run['problem_usage_key']}:
```

Comment:
@pdpinch Currently I am showing `problem_usage_key` as name is not available in model. Let me know what should be displayed here.

Reply:
Sorry, I don't think I understand. 

Do we need a migration or something?

Reply:
Ideally we would show the actual problem name here, but Tasawer is saying that we aren't saving that in our model. I need to confirm that, but with a problem usage ID we should be able to pull the name from the XBlock in edX. I think I might have a code snippet that does that, but I'll finish the first round of this review before I go looking

Reply:
@gsidebo I find this method [get_problem_by_id](https://github.com/mitodl/rapid-response-xblock/blob/master/tests/utils.py#L150) but didn't find any name for problems.

Reply:
@tasawernawaz Take a look at this gist: https://gist.github.com/gsidebo/469d9d2089cf8258b6b1f8235d52c600. The `get_problem_display_names` function takes a list of block id's and returns the display name for that problem. I'm not sure it's the absolute most efficient way to do so in Open edX but it seems to get the job done quickly enough. Those helper functions might make sense to add to `/lms/djangoapps/instructor/views/tools.py`.

I also think it would be a good idea to include the time of the run in the UI. For example "Math Problem #1 - 10:05:15 AM".

Reply:
@gsidebo thanks for providing these util methods, I am able to get display name in a simpler way.